datemath/helpers.py

```diff
@@ -101,15 +106,16 @@ def parse(expression: str, now: Any = None, tz: str = 'UTC', type: Any = None, r
         if debug: print("parse() - will now convert tz to {0}".format(tz))
         now = now.to(tz)
 
+    expression = str(expression)
     if expression == 'now':
         if debug: print("parse() - Now, no dm: {0}".format(now))
         if type:
-            return getattr(now, type)
+            return cast(Arrow, getattr(now, type))
```

Comment:
In strict mode, mypy will whine about returning `Any` from a typed function.
This can either be cast or type-ignored to accept the "type unsafety".